intel_extension_for_transformers/neural_chat/models/base_model.py

```diff
@@ -100,6 +100,9 @@ def load_model(self, kwargs: dict):
         """
         self.model_name = kwargs["model_name"]
         self.device = kwargs["device"]
+        self.use_hpu_graphs = kwargs["use_hpu_graphs"]

```

Comment:
please consider how to organize the device related parameters instead of listing all the parameters

Reply:
ok